client/app/styles/modules/_m-actions-milestones.scss

```diff
@@ -10,9 +10,16 @@
 
 }
 
+.milestone-header {
+  position: relative;
+  box-shadow: inset rem-calc(28) 0 0 $body-background, inset rem-calc(30) 0 0 rgba(0,0,0,0);
+  cursor: pointer;
+}
+
 .milestone {
   position: relative;
   box-shadow: inset rem-calc(28) 0 0 $body-background, inset rem-calc(30) 0 0 $medium-gray;
+  margin-left: 7px;
```

Comment:
Not sure this is gonna work very well on mobile

Reply:
Agreed. I started spinning a bit on how to style this -- I sensed this is djanky but didn't want to spend too much time on it for now since I also sensed it might keep changing